tests/perf/cypress/e2e/Graphload.cy.js

```diff
@@ -0,0 +1,58 @@
+describe('graphload', () => {
+
+    http://localhost:3000/console/graph/namespaces/
+        // ?traffic=grpc%2CgrpcRequest%2Chttp%2ChttpRequest%2Ctcp%2CtcpSent
+        // &graphType=versionedApp
+        // &namespaces=bookinfo%2Ccert-manager
+        // &duration=60
+        // &refresh=15000
+        // &layout=kiali-dagre
+        // &namespaceLayout=kiali-dagre
+
+    var graphUrl;
+
+    before(() => {
+        cy.fixture('graphParams')
+            .then(function (data) {
+                graphUrl = encodeURI("/console/graph/namespaces?traffic="+data.traffic
+                    +"&graphType="+data.graphType
+                    +"&namespaces="+data.namespaces
+                    +"&duration="+data.duration
+                    +"&refresh="+data.refresh
+                    +"&layout="+data.layout
+                    +"&namespaceLayout="+data.namespaceLayout);
+            })
+            .as('data');
```

Comment:
We won't use a fixture when running the perf tests against a real environment.

Reply:
Ah I missed the fact that the fixture was being used to parametrize the test. That's good. We probably won't need to configure all the request params and some of these can be hardcoded for now. The options that will need to be tunable are: https://github.com/kiali/kiali/issues/5144. We'll probably need to do some cleanup between tests and it makes sense to keep that within cypress. Otherwise the person running the tests will need to execute the tests one at a time and manually cleanup resources between them.

Reply:
Ok. 
Is there any other test to add in order to measure any other page load?

Reply:
For now, no, that's the main thing to test but you can also add measuring the overview page in a similar way.

Reply:
Thanks, @nrfox ! Changes added.